### PR TITLE
add azure pipelines config files and tweak the README a bit.

### PR DESCRIPTION
**Pull request by jkwatson**

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# PR + Master build
+name: PR Build for New Relic OpenTelemetry Java Exporters
```
Comment (jasonjkeller) [inline azure-pipelines.yml]: Name needs to be updated for this project?

**Review by jasonjkeller — CHANGES_REQUESTED** [review]
Small naming tweak, otherwise looks good

File: azure-pipelines.yml (hunk 1 above)
Comment (jkwatson) [reply azure-pipelines.yml]: details details.  ;)

**Comment by jkwatson** [comment]
change made

**Review by jasonjkeller — APPROVED** [review]

**Pull request merged**